tests/integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -54,10 +54,6 @@ else
 	)
 fi
 
-if [ ${KATA_HYPERVISOR} == "qemu-sev" ]; then
```

Comment:
@UnmeshDeodhar could you send this in a separate commit? i.e. the the tests enablement and runtimeclasses changes in two different commits as they aren't related.

Reply:
@wainersm changed it. Thanks!